### DISCUSSION TITLE
希望增加像ac86u 的armhnd 的release

### DISCUSSION THREAD

**Discussion by pasicopan** [messense/aliyundrive-webdav#158] [Q&A] [answered]
希望增加像ac86u 的armhnd 的release

**Comment by messense** [comment, marked as answer]
就是 https://github.com/messense/aliyundrive-webdav/releases/download/v0.5.5/aliyundrivewebdav-merlin-arm384-aarch64.tar.gz ？

**Reply by pasicopan** [reply]
ac86u 固件是arm386 吧，另外根据https://github.com/koolshare/armsoft
arm384 和armhnd 对应的linux 内核也不一样。

我是新人只能从命名上看出区别，不确定事实上是否合适。

**Reply by messense** [reply]
不确定就试一下。

**Reply by pasicopan** [reply]
测试了，没问题。。。可以关闭了
